assets/javascripts/discourse/connectors/topic-footer-main-buttons-before-create/assign-button.js.es6

```diff
@@ -3,7 +3,7 @@ import { ajax } from 'discourse/lib/ajax';
 
 export default {
   shouldRender(args, component) {
-    const needsButton = component.currentUser && component.currentUser.get('staff');
+    const needsButton = component.currentUser && component.currentUser.get('staff') && component.siteSettings.assign_enabled;
```

Comment:
If the plugin is disabled, connectors are not supposed to be included. Did you find that they were?

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/lib/plugin-connectors.js.es6#L29